mne/utils/check.py

```diff
@@ -700,3 +700,32 @@ def _suggest(val, options, cutoff=0.66):
         return ' Did you mean %r?' % (options[0],)
     else:
         return ' Did you mean one of %r?' % (options,)
+
+
+def _on_missing(on_missing, msg):
+    """Raise error or print warning with a message.
+
+    Parameters
+    ----------
+    on_missing : 'raise' | 'warn' | 'ignore'
```

Comment:
```suggestion
    on_missing : 'raise' | 'error' | 'warn' | 'warning' | 'ignore'
```

Reply:
+1 on the current PR, -1 on this suggestion